switches/http2/http/HttpDispatcher.h

```diff
@@ -98,6 +99,9 @@ namespace Aseba
 		typedef std::map<HttpMethod, URIHandlerMap> HandlersMap;
 		//! Handlers for known method + URI couples
 		HandlersMap handlers;
+		//! Map of method, split URI to JSON-formatted OAS documentation
+		typedef std::map<HttpMethod, std::map<strings, json>> ApidocsMap;
```

Comment:
One can use the same handler for more than one method, as the method+path is the key. I just propose to use both the handler and its documentation as value. The same handler for a different path would have a different documentation I guess. Currently there are two exactly parallel maps.